### DISCUSSION TITLE
[Esame 6 Settembre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#370] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/06cf5c6c-91e1-4fdf-a925-03faa5d24812)

**Comment by Frisbon** [comment]
> ### Soluzione (sbagliata)

![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/53544818-ce4c-479a-aa46-7b2e079a075b)
